_pages/meetings.md

```diff
@@ -14,14 +14,13 @@ Meetings and telecon times are available as a [Google calendar](https://calendar
 
 ### Community Meetings
 
-The [2021 Spring Python in Heliophysics Community Meeting]({% link
-_pages/meetings/spring2021.md %}) **will be held remotely** on Zoom (see details on the meeting's web page), spanning over the course of four days. The meeting will begin on Monday, May 10th, 2021 and end Thursday, May 13th, 2021.
-* [Registration](https://docs.google.com/forms/d/1tk9uQTm9TzwNV8jle3QCg8IZkhZsQjLKMQDN-02a4IY/edit?usp=sharing)
+[2021 Spring Python in Heliophysics Community Meeting]({% link
+_pages/meetings/spring2021.md %}), May 10–13 (remote).
 * [Agenda, Presentations, Organization Spreadsheets, and Documents](https://drive.google.com/drive/u/0/folders/1HcIQRnVmEXiTgNVx7cVL5mMySxVbUFYc)
 * [Meeting Report](https://docs.google.com/document/d/1G6Gr569NQ_j5FrW3fQkN-QtKazNznsDfHg39SOvnqSc/edit?usp=sharing)
 
 [2020 Fall Python in Heliophysics Community Meeting]({% link
-_pages/meetings/fall2020.md %}), October 26-November 16 (remote).
+_pages/meetings/fall2020.md %}), October 26–November 16 (remote).
```

Comment:
Oh yeah, don't mind me, just changing all the hyphens to en dashes because that's technically how you punctuate intervals. Lol